Modules/_io/winconsoleio.c

```diff
@@ -556,14 +556,18 @@ read_console_w(HANDLE handle, DWORD maxlen, DWORD *readlen) {
     Py_BEGIN_ALLOW_THREADS
     DWORD off = 0;
     while (off < maxlen) {
-        DWORD n, len = min(maxlen - off, BUFSIZ);
+        DWORD n = (DWORD) - 1; 
```

Comment:
It's a style nit, but the spaces around the `-` here (and elsewhere) makes all of these much harder to read - they look like subtractions, rather than a cast. Please write them as `(DWORD)-1`